lib/generators.nix

```diff
@@ -426,4 +426,59 @@ ${expr "" v}
       abort "generators.toDhall: cannot convert a null to Dhall"
     else
       builtins.toJSON v;
+
+  /*
+   * Translate a simple Nix expression to Lua representation with occasional
+   * Lua-inlines that can be construted by mkLuaInline function.
+   *
```

Comment:
in lib/attrsets.nix you can see how some functions are documented. I like those but also I guess that's what is used by https://noogle.dev/ to deduce the type so adding a `Type: AttrSet -> Any -> String` could be interesting.

Reply:
Having that type signature will not help for discovery this generator like in case of primitive functions. No other generators have it. And documentation for [`toINI` in noogle](https://noogle.dev/?selected=%22lib.generators.toINI%22&term=%22toINI%22)  skewed.  
So it also requires changing how block comment formatted as those `*` at beginning interpreted as list item.

Are there any documentation and/or discussion about format?

P.S. It would be great if doctest-like thing will be in Nix.